pkg/controllers/user/logging/utils/testwrap.go

```diff
@@ -429,7 +431,7 @@ func getIndex(dateFormat, prefix string) string {
 	case "YYYY-MM":
 		index = fmt.Sprintf("%s-%d-%d", prefix, today.Year(), today.Month())
```

Comment:
Don't you thing we need to change the month formant here too?

Reply:
We change it because we want to consist with Elasticsearch, for example, April we should use 04 as elasticsearch instead of 4.